### PR TITLE
Pocket twice weekly active user (TWAU) import

### PR DESCRIPTION
**Pull request by jklukas**
For 2022 KPIs

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/pocket/twice_weekly_active_user_counts/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+friendly_name: Twice Weekly Active User Counts
+description: |-
+  Rolling count of twice weekly active users (T-WAU) for Pocket.
```
Comment (kirill-demtchouk) [inline sql/moz-fx-data-shared-prod/pocket/twice_weekly_active_user_counts/metadata.yaml:3]: This actually isn't rolling, but measured each calendar week.

File: sql/moz-fx-data-shared-prod/pocket_derived/twice_weekly_active_user_counts_history_v1/schema.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+fields:
+- mode: NULLABLE
+  name: submission_date
+  type: DATE
+  description: Date of the MAU export from pocket
+- mode: NULLABLE
+  name: active_at
+  type: DATE
+  description: Activity date represented by this row
```
Comment (kirill-demtchouk) [inline sql/moz-fx-data-shared-prod/pocket_derived/twice_weekly_active_user_counts_history_v1/schema.yaml:9]: "Calendar week represented by this row"

**Comment by jklukas** [comment]
Kirill indicates that starting tonight there will be a new column `yearly_cumulative_user_count` that we should account for here.

**Review by kirill-demtchouk — APPROVED** [review]
👍 Looks good! I verified that the data in this new table is consistent with the data on the Pocket side.

**Review by scholtzan — APPROVED** [review]

File: sql/moz-fx-data-shared-prod/pocket_derived/twice_weekly_active_user_counts_v1/query.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+SELECT
+  active_at,
+  user_count,
+  yearly_cumulative_user_count,
+FROM
+  twice_weekly_active_user_counts_history_v1
+WHERE
+  submission_date = @submission_date
```
[the remark this answers is not in the document]
Comment (jklukas) [reply sql/moz-fx-data-shared-prod/pocket_derived/twice_weekly_active_user_counts_v1/query.sql:8]: Correct. This table is always supposed to contain just the most recent partition of the associated `history` table.

[the remark this answers is not in the document]
Comment (kirill-demtchouk) [reply sql/moz-fx-data-shared-prod/pocket_derived/twice_weekly_active_user_counts_v1/query.sql:8]: There are no issues with that from my perspective. The entire thing gets dumped to GCS every night.

**Pull request merged**